dhis-2/dhis-support/dhis-support-test/src/main/resources/tracker/simple_metadata.json

```diff
@@ -18,6 +18,25 @@
       "trackedEntityTypeAttribute": true,
       "unique": true,
       "valueType": "TEXT"
+    },
+    {
+      "id": "i57a0734128",
+      "categoryOptionAttribute": true,
+      "categoryOptionComboAttribute": true,
+      "dataElementAttribute": true,
+      "name": "idScheme related attribute no metadata should have",
```

Comment:
does it make sense to rename this to
`idScheme related attribute without metadata indentifier`?

Reply:
That name proposal does not sound correct to me 🤔 The attribute itself has no metadata identifier. It's that other metadata like a program can have an attribute value for that attribute i.e. an identifier for the idScheme `ATTRIBUTE:i57a0734128`.

Reply:
ok, it's just when I read `idScheme related attribute no metadata should have`, I don't know that that means